openmeetings-web/src/main/java/org/apache/openmeetings/web/room/raw-video.js

```diff
@@ -391,7 +391,7 @@ var Video = (function() {
 		_resizeDlgArea(hasVideo ? size.width : 120
 			, hasVideo ? size.height : 90);
 		if (hasVideo && !isSharing && !isRecording) {
-			VideoUtil.setPos(v, VideoUtil.getPos(VideoUtil.getRects(VIDWIN_SEL), sd.width, sd.height + 25));
+			VideoUtil.arrange();
```

Comment:
Imagine user moved video windows to preferred position
then someone have restarted his/her video device or exit the room
and all user actions will be reset ....

I believe these 3 calls to `arrange` shoul be removed

Reply:
What if I won't do full rearrangement in "default" mode only? Will it be acceptable solution?

Reply:
I believe this would be OK
IF the only way to switch the mode will be `Ctrl+Shift+X` i.e. mode will not be switched if windows were re-arranged by hotkey

Reply:
Sure. I can rework it so in 0 mode (default) all currently present arrange hotkeys and window arrangements will behave exactly like they are behave now and wouldn't switch to different mode.

About ways to switch mode I'm planning to add settings to room configuration in follow-ups patches. For ability to set different default mode and change resolutions for modes 1 & 2. So they are not hardcoded and can be changed on per room basis. And also add global settings to what set them on automatic room creation.